pkg/session_recording/recordings.jsx

```diff
@@ -543,14 +524,11 @@
         }
 
         render() {
-            const dateSince = this.state.dateSince;
-            const dateUntil = this.state.dateUntil;
-
             if (this.state.recordingID === null) {
                 return (
                     <RecordingList
-                        dateSince={dateSince} onDateSinceChange={this.handleDateSinceChange}
-                        dateUntil={dateUntil} onDateUntilChange={this.handleDateUntilChange}
+                        onDateSinceChange={this.handleDateSinceChange}
+                        onDateUntilChange={this.handleDateUntilChange}
```

Comment:
I will check how to make it. It depends on where actual URL will be "parsed". I will figure out, but I did this commit to clarify and removed parts which are actually not required.
It's possible that I might put those vars back when I get to URL work.